src/editor/index.js

```diff
@@ -41,68 +28,61 @@ class PopupSidebar extends Component {
-					<PanelBody title={ __( 'Popup Triggers' ) } initialOpen={ true }>
-						<RadioControl
-							label={ __( 'Trigger' ) }
-							help={ __( 'The event to trigger the popup' ) }
-							selected={ trigger_type }
-							options={ [
-								{ label: __( 'Timer' ), value: 'time' },
-								{ label: __( 'Scroll Progress' ), value: 'scroll' },
-							] }
-							onChange={ value => onMetaFieldChange( 'trigger_type', value ) }
-						/>
-						{ 'time' === trigger_type && (
-							<RangeControl
-								label={ __( 'Delay (seconds)' ) }
-								value={ trigger_delay }
-								onChange={ value => onMetaFieldChange( 'trigger_delay', value ) }
-								min={ 0 }
-								max={ 60 }
-							/>
-						) }
-						{ 'scroll' === trigger_type && (
-							<RangeControl
-								label={ __( 'Scroll Progress (percent) ' ) }
-								value={ trigger_scroll_progress }
-								onChange={ value => onMetaFieldChange( 'trigger_scroll_progress', value ) }
-								min={ 1 }
-								max={ 100 }
-							/>
-						) }
-						<SelectControl
-							label={ __( 'Frequency' ) }
-							value={ frequency }
-							onChange={ value => onMetaFieldChange( 'frequency', value ) }
-							options={ [
-								{ value: 0, label: __( 'Once per user' ) },
-								{ value: 5, label: __( 'Every 5 page views' ) },
-								{ value: 25, label: __( 'Every 25 page views' ) },
-								{ value: 100, label: __( 'Every 100 page views' ) },
-							] }
-						/>
-						<SelectControl
-							label={ __( 'Placement' ) }
-							value={ placement }
-							onChange={ value => onMetaFieldChange( 'placement', value ) }
-							options={ [
-								{ value: 'center', label: __( 'Center' ) },
-								{ value: 'top', label: __( 'Top' ) },
-								{ value: 'bottom', label: __( 'Bottom' ) },
-							] }
-						/>
-					</PanelBody>
-				</PluginSidebar>
+				<RadioControl
+					label={ __( 'Trigger' ) }
+					help={ __( 'The event to trigger the popup' ) }
+					selected={ trigger_type }
+					options={ [
+						{ label: __( 'Timer' ), value: 'time' },
+						{ label: __( 'Scroll Progress' ), value: 'scroll' },
+					] }
+					onChange={ value => onMetaFieldChange( 'trigger_type', value ) }
+				/>
+				{ 'time' === trigger_type && (
+					<RangeControl
+						label={ __( 'Delay (seconds)' ) }
+						value={ trigger_delay }
+						onChange={ value => onMetaFieldChange( 'trigger_delay', value ) }
+						min={ 0 }
+						max={ 60 }
+					/>
+				) }
+				{ 'scroll' === trigger_type && (
+					<RangeControl
+						label={ __( 'Scroll Progress (percent) ' ) }
+						value={ trigger_scroll_progress }
+						onChange={ value => onMetaFieldChange( 'trigger_scroll_progress', value ) }
+						min={ 1 }
+						max={ 100 }
+					/>
+				) }
+				<SelectControl
+					label={ __( 'Frequency' ) }
+					value={ frequency }
+					onChange={ value => onMetaFieldChange( 'frequency', value ) }
+					options={ [
+						{ value: 0, label: __( 'Once per user' ) },
```

Comment:
I think you may need to manually update either this PR or #7 after one of these two PRs is merged, because I don't think git will be smart enough to recognize [this frequency change moved files](https://github.com/Automattic/newspack-popups/pull/7/files#diff-981bcddd6b14e574830a9e9c55112765R81). Just a heads-up.